docs/developer/release-cycle.md

```diff
@@ -0,0 +1,98 @@
+* Week 15-16: RC
+    * The release branch is created
+    * Weekly RC releases will be released based on the release branch
+        * RC releases are as close as possible to the final release but they are still undergoing further testing
+    * Development of the next release can technically start on the main branch, but some changes might be delayed
+      to ensure easier cherry-picking of other changes to the release branch.
+    * There is a feature freeze on the release branch
+        * Only cherry-picks for documentation, bug fixes, test fixes and test additions are allowed
+* Week 17: GA
+    * `.0` GA release is created based on the release branch
+* After that:
+    * `.1+`: Monthly patch releases will be released based on the release branch
+    * Cherry-picks to the release branch are allowed according to the backport policy
+    * Providers create releases using the new CAPI minor release when they are ready
+    * Development of the next release can now officially start on the main branch
+
+Some additional notes:
+
+* Support for new Kubernetes minor versions (for management and workload clusters) is first implemented
+  on the main branch, then cherry-picked into supported release branches when feasible and eventually
+  released in the next monthly patch release(s).
+    * **Note**: We usually don't have to bump Go dependencies to support new Kubernetes minor versions as it's not necessary
+      to run on a management cluster with the new version or create a workload cluster with the new version.
+      If it becomes necessary to bump dependencies to a new CR/Kubernetes minor version, the change cannot be cherry-picked
+      as bumping dependencies to a new minor version is a breaking change.
+* New Kubernetes and Controller-Runtime (CR) minor releases are only picked up on the main branch.
+
+The following diagram visualizes the release cycles with the v1.2, v1.3 and v1.4 releases:
+
+<!-- The release cycle png can be opened and edited in draw.io -->
+![Release Cycle Overview](release-cycle-overview.png)
+
+# v1.3 Release Cycle timeline
+
+**Note**: This release will be used to set up documentation, automation, etc. for the first release team which will start with
+the `v1.4` release cycle.
+
+The following table shows the preliminary dates for the `v1.3` release cycle.
+
+| **What**                                       | **When**                   | **Week** |
+|------------------------------------------------|----------------------------|----------|
+| Start of Release Cycle                         | Monday 18th July 2022      | week 1   |
+| 1.3.0-beta.0 released                          | Tuesday 1st November 2022  | week 13  |
+| 1.3.0-beta.1 released                          | Tuesday 8th November 2022  | week 14  |
+| release-1.3 branch created (**[Code Freeze]**) | Tuesday 15th November 2022 | week 15  |
```

Comment:
nit, to make it consistent with the other table: 
```suggestion
| release-1.3 branch created (**Begin [Code Freeze]**) | Tuesday 15th November 2022 | week 15  |
```

Reply:
Thx!